Runtime/Behaviors/PlayAudioBehavior.cs

```diff
@@ -12,6 +12,7 @@ namespace Innoactive.Creator.Core.Behaviors
     /// A behavior that plays audio.
     /// </summary>
     [DataContract(IsReference = true)]
+    [HelpLink("https://developers.innoactive.de/documentation/creator/latest/articles/innoactive-creator/default-behaviors.html#audioplay-texttospeech-audio")]
```

Comment:
This condition is used for Play Audio and TTS, just with different MenuItems. So unsure what to do 🤔